starboard/linux/x64x11/toolchain/BUILD.gn

```diff
@@ -13,12 +13,19 @@
 # limitations under the License.
 
 import("//build/config/clang/clang.gni")
+import("//starboard/build/toolchain/cobalt_toolchains.gni")
 import("//starboard/shared/toolchain/overridable_gcc_toolchain.gni")
 
 overridable_clang_toolchain("starboard") {
   clang_base_path = clang_base_path
 }
 
+cobalt_clang_toolchain("cobalt") {
+  variables = {
```

Comment:
Is this block necessary? Why not keep the variables in the root?

Reply:
i think the idea is to keep the scope of all these variables limited using forward_variables_from(invoker.variables,

so gcc_toolchain only ingests what's defined in the `variables` scope instead of everything ("*")